src/jit/gentree.h

```diff
@@ -2429,6 +2524,14 @@ struct GenTreeCall final : public GenTree
     }
 #endif // !LEGACY_BACKEND
 
+    // Returns true if the call has retBuf argument
+    bool HasRetBufArg() const { return (gtCallMoreFlags & GTF_CALL_M_RETBUFFARG) != 0; }
```

Comment:
I am not sure of it though.  A call doesn't return RetBuf (exept for x64 unix) as it return value.  The return type of a call node returning a struct through RetBufArg is TYP_VOID, besides retBufArg is an explicit arg of CALL node.  One can argue it makes sense to have this method on call node.